src/main/java/com/google/codeu/data/Message.java

```diff
@@ -18,14 +18,28 @@
 
 import java.util.UUID;
 
-/** A single message posted by a user. */
+/** The sting recipient helps to display messages received from another user. */
 public class Message {
 
+  private String recipient;
+  public Message(String user, String text, String recipient) {
+  this(UUID.randomUUID(), user, text, System.currentTimeMillis(), recipient);
+}
+
+public Message(UUID id, String user, String text, long timestamp, String recipient) {
```

Comment:
Here you've added two brand new constructors with the new recipient field we want. What we actually want is to *change* the constructors that already exist (lines 47 to 56).

Reply:
Resolved. Thank you 😄